packages/gafl-webapp-service/src/handlers/agreed-handler.js

```diff
@@ -149,7 +149,9 @@ const processPayment = async (request, transaction, status) => {
     status[COMPLETION_STATUS.paymentCompleted] = true
     await request.cache().helpers.status.set(status)
 
-    await request.ga.ecommerce().purchase(getTrackingProductDetailsFromTransaction(transaction))
+    await request.ga
+      .ecommerce()
+      .purchase(getTrackingProductDetailsFromTransaction(transaction), transaction.payment.payment_id, process.env.CHANNEL)
```

Comment:
Could you update the affiliation to match the current service please?  It uses the following values:

- Get a fishing licence service - Web sales
- Get a fishing licence service - Telephone sales